src/http-libwebsockets.c

```diff
@@ -537,7 +650,7 @@ static int callback_metrics(struct lws *wsi, enum lws_callback_reasons reason,
 {
     qd_http_server_t *hs = wsi_server(wsi);
     stats_t *stats = (stats_t*) user;
-    uint8_t buffer[LWS_PRE + 2048];
+    uint8_t buffer[LWS_PRE + 26326];
```

Comment:
magic number is magic :D

Reply:
please explain magic number, consider making it an int constant

Reply:
@jiridanek finally updated as suggested. Please feel free to review.